lib/config.js

```diff
@@ -27,13 +27,13 @@ function getInCluster () {
       ' and KUBERNETES_SERVICE_PORT must be defined');
   }
 
-  const cert = fs.readFileSync(certPath, 'utf8');
+  const ca = fs.readFileSync(certPath, 'utf8');
```

Comment:
Good catch!

I was wondering about the `my-cert` string in the test, but just left it alone -- tell me if that should be changed to `my-ca`, too.